students/scott/session01/break_me.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+
+# Write a function that generates a NameError
+
+def generate_name_error():
+	print(paul_blart)
+
+generate_name_error()
+
+
+
+
+#Write a function that generates a TypeError
+
+a = 1
+b = "threads"
+
+this_thing = a + b
+
+def generate_type_error():
+	print("this_thing")
+
+generate_type_error()
+
+
+
+
+#Write a function that generates a SyntaxError
+
+def generate_syntax_error():
+#	print)"foo" and "bar")
+
+generate_syntax_error()
+
```

Comment:
The print line in your generate_syntax_error does cause a syntax error. Most syntax errors are found on loading, not by running, so unlike the other errors that are in functions, it is not sufficient to comment out the line where you are calling the function, but the actual line where the syntax error is.